phone/src/os/phone/hooks/useLocalStorage.ts

```diff
@@ -25,16 +27,31 @@ function useLocalStorage<T>(key: string, initialValue: T) {
     }
   });
 
+  const getStorageItem = (localKey: string) => {
+    try {
+      const data = window.localStorage.getItem(UNIQ_PREFIX + localKey);
+      // return stored or initValue if none stored
+      return data ? JSON.parse(data) : initialValue;
+    } catch (e) {
+      // TODO: Integrate debug error state here
+      console.log(e);
+      // We still return initVal if error is thrown
+      return initialValue;
+    }
+  };
+
   // Trying to rep useState setter pretty close just with localStorage
-  const setVal = (value: T | ((val: T) => T)) => {
+  const setVal = (localKey, value: T | ((val: T) => T)) => {
+    console.log(localKey, value);
```

Comment:
Don't have console.log's integrated into a master merge, it gets too messy. I *eventually* want to work on adding a logging class where we can more easily capture logs at different verbosity levels.

Reply:
If need be add a console.log that is commented but I would prefer not to.